ReactWindows/ReactNative/CoreModulesPackage.cs

```diff
@@ -0,0 +1,75 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using ReactNative.Tracing;
+using ReactNative.UIManager;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace ReactNative
+{
+    /// <summary>
+    /// Package defining core framework modules (e.g., <see cref="UIManagerModule"/>). 
+    /// It should be used for modules that require special integration with
+    /// other framework parts (e.g., with the list of packages to load view
+    /// managers from).
+    /// 
+    /// TODO:
+    /// 1. Add Core native modules
+    /// 2. Implement UIManagerModule
+    /// 3. Add remaining JavaScript modules
+    /// </summary>
+    class CoreModulesPackage : IReactPackage
```

Comment:
We could consider creating an IReactPackage base class that did some of the type checking that we do in JavaScriptModulesConfig.  We don't have the Class<? extends T> syntax in C#...